.github/workflows/on-pull-request.yaml

```diff
@@ -38,3 +38,52 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           fail_ci_if_error: true
+  e2e-tests:
+    name: Run e2e tests
+    runs-on: ubuntu-latest
```

Comment:
There is enough in this e2e job that putting it in its own workflow file starts to make sense.

Also, if you want to set an ENVVAR to be used across all of the steps, you can do it here.  For example:
```yml
  e2e-tests:
    name: Run e2e tests with cypress
    runs-on: ubuntu-latest
    env:
      FORKLIFT_IMAGE_REPOSITORY: localhost:5001
```